app/controllers/my_module_comments_controller.rb

```diff
@@ -76,7 +76,9 @@ def create
                 comment: @comment
               }
             ),
-            date: @comment.created_at.strftime('%d.%m.%Y')
+            date: @comment.created_at.strftime('%d.%m.%Y'),
+            linked_id: @my_module.id,
```

Comment:
I would just maybe add comment to those 2 rows so future devs will know the returned data is used for counter badge. E.g.
```ruby
linked_id: @my_module.id, # Used for counter badge
counter: @my_module.task_comments.count # Used for counter badge
```